src/components/ClusterOverview/Inventory/Inventory.js

```diff
@@ -1,80 +1,66 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Col, Icon } from 'patternfly-react';
 
-import { getNodeErrorStatuses } from './nodeStatus';
 import {
   DashboardCard,
   DashboardCardBody,
   DashboardCardHeader,
   DashboardCardTitle,
   DashboardCardTitleHelp,
 } from '../../Dashboard/DashboardCard';
-import { ClusterOverviewContextGenericConsumer } from '../ClusterOverviewContext';
+import { ClusterOverviewContext } from '../ClusterOverviewContext';
+import { mapNodesToProps, mapPodsToProps, mapPvcsToProps, mapVmsToProps } from './utils';
+import { InventoryRow } from './InventoryRow';
 
-const InventoryItemStatus = ({ item }) => {
-  let ok = true;
-  if (item.kind === 'Node') {
-    ok = !item.data.some(i => getNodeErrorStatuses(i).length > 0);
-  }
-  return ok ? (
-    <div className="kubevirt-inventory__item-status">
-      <Icon type="fa" name="check-circle" size="2x" className="kubevirt-inventory__item-status-icon" />
-      <span>{item.data.length}</span>
-    </div>
-  ) : (
-    <div>
-      <span className="glyphicon glyphicon-remove-circle" />
-    </div>
-  );
-};
+const InventoryBody = ({ nodes, pods, vms, vmis, pvcs, migrations }) => (
+  <React.Fragment>
+    <InventoryRow title="Nodes" {...mapNodesToProps(nodes)} />
```

Comment:
this is not generic enough. Other overviews will want to show different resources here. Inventory card should accept something like `items` and every item would define title and its statuses.

Reply:
this component is not meant to be generic, but an implementation of a card in clusterOverview dashboard.

If someone wishes to have inventory card in their own dashboard, they can create their card with their own InventoryRows. This should be simple and generic enough and can also be enhanced if something is missing.

Reply:
In that case, we should also remove `ClusterDetails` and use `Details` only. If anybody will want to use `Details` card they will implement their own with their `DetailItems`. I do not mind this approach as long as its consistent among all cards.

Reply:
Ok, I will refactor it. I would rather do it in the follow up as this PR is already packed.

Reply:
+1